runtime/parachains/src/inclusion_inherent.rs

```diff
@@ -202,6 +202,10 @@ impl<T: Config> ProvideInherent for Module<T> {
 					{
 						Call::inclusion(signed_bitfields, backed_candidates, parent_header)
 					} else {
+						sp_runtime::print(
+							"WARN: dropping signed_bitfields and backed_candidates because they produced \
```

Comment:
If there's a better way to emit a log message from the runtime, I'd be glad to learn it.

Reply:
it would've been great if we printed the error it outputs: that would allow us to tell what kind of error happened

I think we can use `frame_support::debug::print!` for that. I am not sure if dispatch error has a proper `Debug` implementation

Reply:
6e5bb037